theme/static/style/style.css

```diff
@@ -248,76 +248,93 @@ pre code {
 
 /* Screenshots Section */
 
-.screenshot-title{
+/* Title */
+.screenshot-title {
     padding: 2% 0;
     text-align: center;
     line-height: 1.4em;
 }
 
-.screenshot-title h3{
+.screenshot-title h3 {
     font-size: 3rem;
 }
 
-.screenshot-title p{
+.screenshot-title p {
     margin-top: 1.5rem;
 }
 
-.img{
+/* Image */
+
+.img-wrapper, .img {
+    width: 100%;
+    height: 30vw;
+}
+
+.img-wrapper {
+    position: relative;
+    padding: 3rem 2rem;
+    margin: auto;
+    white-space: nowrap;
+    overflow: hidden;
+}
+
+.img {
     position: absolute;
-    top: 5rem;
     display: block;
-    width: 922px;
-    height: 450px;
-    box-shadow: 0 0.5rem 1rem 0 rgba(0,0,0, .3);
-    background-size: cover;
+    background-size: contain;
+    background-position: center;
+    background-repeat: no-repeat;
 }
```

Comment:
ATM from what I can see the images look resized and much smaller on mobile. Definitely has something to do with these changes but I'm not quite certain which one exactly.